### PR TITLE
feat(NumberFormat): add monospace support

### PR DESCRIPTION
**Pull request by AadneRo**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/number-format/properties.mdx [hunk 1]
```diff
@@ -30,6 +30,7 @@ redirect_from:
...
 | `link`                                  | _(optional)_ use `tel` or `sms` to enable a clickable / touchable anchor link.                                                                                                                                                                                                                                                                                                                        |
+| `monospace`                             | _(optional)_ Sets the font to [DNB Mono Regular](/quickguide-designer/fonts/#dnbmono-regular)                                                                                                                                                                                                                                                                                                         |
```
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/components/number-format/properties.mdx:33]:
```suggestion
| `monospace`                             | _(optional)_ Sets the font to [DNB Mono Regular](/quickguide-designer/fonts/#dnbmono-regular).                                                                                                                                                                                                                                                                                                         |
```

**Review by langz — APPROVED** [review]
I think it looks great 💯

**Review by tujoworker — APPROVED** [review]
Yes, nice work 👍

**Pull request merged**